website/docs/deploy/configuring-unleash.md

```diff
@@ -254,57 +258,55 @@ If you want to read content from a file, you should either initialize Unleash vi
 
 <TabItem value="js" label="JavaScript" default>
 
-``` js title="Reading from the file system in JavaScript"
+```js title="Reading from the file system in JavaScript"
 const unleashOptions = {
-    db: {
-        // other options omitted for brevity
-        ssl: {
-            ca: fs.readFileSync('/path/to/server-certificates/root.crt').toString(),
-        }
-    }}
+  db: {
+    // other options omitted for brevity
+    ssl: {
+      ca: fs.readFileSync('/path/to/server-certificates/root.crt').toString(),
+    },
+  },
+};
 ```
 
 </TabItem>
 
 <TabItem value="env" label="Environment variables (bash)">
 
-``` bash title="Reading from the file system with bash"
+```bash title="Reading from the file system with bash"
 DATABASE_SSL="{ \"key\": \"$(cat /path/to/server-certificates/root.crt)\" }"
 ```
 
 </TabItem>
 
 </Tabs>
 
-
-
 ### Enabling self-signed certificates
 
-
 To use self-signed certificates, you should set the SSL property `rejectUnauthorized` to `false` and set the `ca` property to the value of the certificate:
 
 <Tabs groupId="db-configuration-options">
 
 <TabItem value="js" label="JavaScript" default>
 
-``` js title="Enable self-signed certificates"
+```js title="Enable self-signed certificates"
 const unleashOptions = {
-    db: {
-        // other options omitted for brevity
-        ssl: {
-            rejectUnauthorized: false,
-            ca: fs.readFileSync('/path/to/server-certificates/root.crt').toString(),
-        }
-    }}
-
+  db: {
+    // other options omitted for brevity
+    ssl: {
+      rejectUnauthorized: false,
+      ca: fs.readFileSync('/path/to/server-certificates/root.crt').toString(),
+    },
+  },
+};
 ```
 
 </TabItem>
 
 <TabItem value="env" label="Environment variables (bash)">
 
-``` bash title="Enable self-signed certificates"
-DATABASE_SSL="{ \"rejectUnauthorized\": false, \"key\": \"$(cat /path/to/server-certificates/root.crt)\" }"
+```bash title="Enable self-signed certifsessione, "key": "$(cat /path/to/server-certificates/root.crt)" }"
```

Comment:
Whoops, I think something happened here. It looks you deleted part of the example but added a little bit extra at the same time? 😅